src/components/rating/_rating.scss

```diff
@@ -18,8 +20,20 @@ $includeHtml: false !default;
     display: flex;
     align-items: center;
     height: $rateHeight;
+    overflow: visible;
```

Comment:
that's the default value?

Reply:
`component` mixin uses `overflow: hidden` as default